pages/common/sphinx-build.md

```diff
@@ -0,0 +1,16 @@
+# sphinx-build
+
+> Generates documentation from the files in sourcedir and places it in the outputdir.
+> More information: http://www.sphinx-doc.org/en/master/man/sphinx-build.html.
+
+- Build documentation as "epub" and put it in "builddir":
```

Comment:
```suggestion
- Build documentation for a Python project:
```
More concise and general, we can already see source dir, build dir and builder in the example (see my next comment)

Reply:
Again, let’s drop the `for a Python project`.